packages/web/components/Modal/BeAReviewerModal/index.js

```diff
@@ -54,6 +61,20 @@ const BeAReviewerModal = ({ closeModal }) => {
 		}
 	}, [categoryValue]);
 
+	// eslint-disable-next-line consistent-return
+	const handleAcceptedTerms = (type) => {
+		const types = Object.keys(acceptedTerms);
+
+		if (!type || !types.some((item) => item === type)) {
+			return null;
+		}
+
+		setAcceptedTerms({
+			...acceptedTerms,
+			[type]: !acceptedTerms[type],
+		});
+	};
+
```

Comment:
```suggestion
	const handleAcceptedTerms = (type) => {
		const types = Object.keys(acceptedTerms);

		if (!type || !types.some((item) => item === type)) {
			return null;
		}

		return setAcceptedTerms({
			...acceptedTerms,
			[type]: !acceptedTerms[type],
		});
	};

```

There's no need to disable this rule, just return